tensorflow/core/api_def/base_api/api_def_Bitcast.pbtxt

```diff
@@ -12,6 +12,40 @@ If `T` is smaller than `type`, the operator requires that the rightmost
 dimension be equal to sizeof(`type`)/sizeof(`T`). The shape then goes from
 [..., sizeof(`type`)/sizeof(`T`)] to [...].
 
+tf.bitcast() and tf.cast() work differently when real dtype is casted as a complex dtype
+(e.g. tf.complex64 or tf.complex128) as tf.cast() make imaginary part 0 while tf.bitcast()
+gives module error.
+For example,
+Example 1:
```

Comment:
Please wrap the code samples in \`\`\`python ... \`\`\` (see 62e25eb change for an example).